src/main/flight/mixer.c

```diff
@@ -450,7 +450,7 @@ void mixerConfigureOutput(void)
     }
 
     // in 3D mode, mixer gain has to be halved - 20180329-teracis: added cli switch for testing
-    if (feature(FEATURE_3D) && (mixerConfig()->ddd_mixergain_halving)) {
+    if (feature(FEATURE_3D) && !(mixerConfig()->mix3d_gain_fix)) {
```

Comment:
The extra parans are not really needed and it could just be `!mixerConfig()->mix3d_gain_fix`, but not a big deal.